engine/apps/api/serializers/alert_receive_channel.py

```diff
@@ -124,15 +125,11 @@ def update(
         if alert_group_labels is None:
             return instance
 
-        # update inheritable labels
-        inheritable_key_ids = [
-            key_id for key_id, inheritable in alert_group_labels["inheritable"].items() if inheritable
-        ]
-        instance.labels.filter(key_id__in=inheritable_key_ids).update(inheritable=True)
-        instance.labels.filter(~Q(key_id__in=inheritable_key_ids)).update(inheritable=False)
-
         # update DB cache for custom labels
         cls._create_custom_labels(instance.organization, alert_group_labels["custom"])
+        # save static labels as integration labels
+        # todo: it's needed to cover delay between backend and frontend rollout, and can be removed later
```

Comment:
can you elaborate a bit more on this? I think I know what you're referring to, but if we rollout the backend changes, before the frontend changes, would that solve the problem stated here?

Reply:
that's the point, when backend changes are rolled out, it will still be possible to create static labels using frontend, since frontend has some delay in rollout, so backend will automatically save them as integration labels instead. Does it make sense?

Reply:
ahh okay, yes, I see what you mean, thanks for clarifying 👍